lib/bump.js

```diff
@@ -109,7 +109,9 @@ module.exports = async (type, preSuffix) => {
 	createSpinner('Creating release commit');
 
 	try {
-		await runGitCommand(`git add -A && git commit -a -m "${version}"`);
+		await runGitCommand(
+			`git add package.json package-lock.json && git commit -m "${version}"`
```

Comment:
This would fail if a `package-lock.json` is missing, i.e. if the user uses yarn.
You should check before if there is a `package-lock.json` or `yarn.lock` present in my opinion.

Reply:
Thank you for the feedback. I've added a check to commit `package-lock.json` only if it is present. I excluded `yarn.lock` from the commit as I see release is only changing package.json and package-lock.json in the `increment()` function.